srcpkgs/ibus-chewing/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'ibus-chewing'
+pkgname=ibus-chewing
+version=1.5.1
+revision=1
+archs="x86_64"
+build_style=gnu-makefile
+hostmakedepends="git cmake pkg-config"
+makedepends="gettext-devel libchewing-devel ibus-devel gob2 libX11-devel gtk+3-devel glib-devel findutils"
+depends="ibus libchewing"
+short_desc="Chewing input method for ibus"
+maintainer="ctoid <funk443@yandex.com>"
+license="GPL-2.0-or-later"
+homepage="https://github.com/definite/ibus-chewing"
+
+do_fetch() {
+	git clone https://github.com/definite/ibus-chewing.git $wrksrc
```

Comment:
We don't generally accept packages using git clone. We definitely will not accept a package using git and cloning straight to the master branch.

Please use tagged releases whenever possible, and use proper distfiles whenever possible.